tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -428,6 +428,12 @@ generate_qemu_options() {
 	# Support Ceph RADOS Block Device (RBD)
 	[ -z "${static}" ] && qemu_options+=(functionality:--enable-rbd)
 
+	# In "passthrough" security mode
+	# (-fsdev "...,security_model=passthrough,..."), qemu uses a helper
+	# application called virtfs-proxy-helper(1) to make certain 9p
+	# operations safer.
+	qemu_options+=(functionality:--enable-virtfs)
+	qemu_options+=(functionality:--enable-attr)
```

Comment:
Ok, the change was reverted in a different commit, so I think this bit needs to go into the earlier one that removed it.